package.json

```diff
@@ -1,6 +1,5 @@
 {
-	"name": "@newfold-labs/wp-module-coming-soon",
```

Comment:
Was this intentionally removed?

Reply:
Yes. https://github.com/newfold-labs/wp-module-coming-soon/pull/27/commits/1a9fb4b1bcb7d68f030ab0453fcd180a8eb83ac0